### PR TITLE
Update for 43.0

### PR DESCRIPTION
**Pull request by bellegarde-c**

### CODE REVIEW

**Comment by bellegarde-c** [comment]
It does not build on flathub. I added network suuport because of this.

```
gnome-online-accounts:

Cloning into 'librest'...
fatal: unable to access 'https://gitlab.gnome.org/gnome/librest.git/': Could not resolve host: gitlab.gnome.org

```

**Comment by dusansimic** [comment]
@bellegarde-c 

> It does not build on flathub. I added network suuport because of this.

You shouldn't need network for that. Flatpak builder clones sources regardless of weather network is enabled in build options or not. That setting affects weather network is enabled during the build step (scripts, build systems, etc.).

**Comment by bellegarde-c** [comment]
@dusansimic Manifest updated to include librest

File: org.gnome.Geary.yaml [hunk 1]
```diff
@@ -239,9 +246,8 @@ modules:
     config-opts:
       - "-Dprofile=release"
     sources:
-      - type: archive
-        url: https://download.gnome.org/sources/geary/40/geary-40.0.tar.xz
```
Comment (MatMaul) [inline org.gnome.Geary.yaml:243]: The 43 release tarball is now available, probably better to keep using that than the git ?

File: org.gnome.Geary.yaml [hunk 2]
```diff
@@ -119,8 +125,8 @@ modules:
       - "-DICAL_GLIB_VAPI=true"
     sources:
       - type: archive
-        url: https://github.com/libical/libical/releases/download/v3.0.11/libical-3.0.11.tar.gz
-        sha256: 1e6c5e10c5a48f7a40c68958055f0e2759d9ab3563aca17273fe35a5df7dbbf1
+        url: https://github.com/libical/libical/releases/download/v3.0.14/libical-3.0.14.tar.gz
```
Comment (MatMaul) [inline org.gnome.Geary.yaml:128]: 3.0.16 is now available.

**Review by MatMaul — COMMENTED** [review]
I did a similar PR before noticing yours...
https://github.com/flathub/org.gnome.Geary/pull/58/files

`gspell` and `gmime` also have updates.

File: org.gnome.Geary.yaml (hunk 1 above)
Comment (bellegarde-c) [reply org.gnome.Geary.yaml:243]: I plan to release fixes to flathub as soon I fix them in Geary but I do not plan to release a sub release as soon I fix a bug.

That's why I use git + branch + commit

Comment (bellegarde-c) [reply org.gnome.Geary.yaml:243]: I plan to release fixes to flathub as soon as  I fix them in Geary but I do not plan to release a sub release as soon I fix a bug.

That's why I use git + branch + commit

Comment (MatMaul) [reply org.gnome.Geary.yaml:243]: Nevermind I just can't read... I should sleep more ^^

**Review by MatMaul — APPROVED** [review]

**Pull request merged**